### ISSUE TITLE
random crashe

### ISSUE THREAD

**Issue by safareli** [avocode/avocode#468]
I had two designs open and was working on other application then  i switched to avocado and it crashed  (btw memory was full #370)
-----
App v.2.4.3-3630f5a
Source UID: 121182
```
Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
  at invariant (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/invariant.js:35:15)
  at Object.getID (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:85:13)
  at Object.ReactMount.isRenderedByReact (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:567:25)
  at Object.ReactMount.getFirstReactDOM (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:582:22)
  at handleTopLevelImpl (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactEventListener.js:63:35)
  at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:134:20)
  at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingStrategy.js:66:19)
  at Object.batchedUpdates (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:109:20)
  at ReactEventListener.dispatchEvent (/Users/safareli/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactEventListener.js:175:20)

```

**Comment by rychlis** [comment]
Thanks for the report, the error is probably caused by full memory and we will need to fix #370 

The problem is that the error reporting doesn't work properly in 2.4.3 minified environment, so we cannot really see what happened. I guess you are not able to reproduce this issue reliably?

**Comment by safareli** [comment]
you are right I can't reproduce this issue and yap it's related to #370 
it crashed and I just reported it if it was helpful and I don't have more info so I'l just close it
